composer.json

```
@@ -21,11 +21,28 @@
   "require": {
     "php": ">=7.2",
```

same for Symfony 5.4 we need php >=7.2.5
```suggestion
    "php": ">=7.2.5",
```

---

Upgraded the PHP version to 7.2.5